lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -9036,11 +9032,23 @@ namespace Js
             length = pArr->length;
         }
 
+        if (length.IsSmallIndex())
```

Comment:
Hi  Taylor, hope things are well. The link you posted doesn't really shed any light on the perf impact to people outside of redmond. could you quantitate the perf improvement and on what testing  you found the regressive behavior?